packages/grafana-ui/src/types/datasource.ts

```diff
@@ -414,6 +414,8 @@ export interface DataQuery {
   datasource?: string | null;
 
   metric?: any;
+
+  expr?: string;
```

Comment:
This is a datasource-specific attribute. Can we do without? Also see other comment when copying `refId` and `expr` only.

Reply:
Ah, I didn't know that. Sure, I'll change this